### PR TITLE
Push dashboard to ECS

### PR DESCRIPTION
**Pull request by OzTK**
The infra was tested on tmp-staging and works with CodePipelines from the account it gets deployed to

### CODE REVIEW

**Comment by OzTK** [comment]
Stack to be re-deployed after this has been deployed to ECR

**Comment by jvmakine** [comment]
👀

File: ci/cloudformation.cf.yml [hunk 1]
```diff
@@ -1,6 +1,15 @@
 AWSTemplateFormatVersion: 2010-09-09
 Description: 'CloudFormation stacks for CodePipeline dashboard'
-
+Parameters:
+  Environment:
```
Comment (jvmakine) [inline ci/cloudformation.cf.yml]: I wonder if this is strictly needed? Presumably we will run only one dashboard per account anyway. Not sure if it gives us much to tag the SG with this.

Comment (OzTK) [reply ci/cloudformation.cf.yml]: We have 2 dashboards already, both tagged with that. They each have their own SG as they operate on different ports. Maybe they should use the same?

Comment (OzTK) [reply ci/cloudformation.cf.yml]: Discussed offline, not necessary, but useful in AWS console for filtering

**Pull request merged**